build/run-tasks.js

```diff
@@ -0,0 +1,151 @@
+const colour = require('nodemon/lib/utils/colour')
+const del = require('del')
+const fs = require('fs')
+const fse = require('fs-extra')
+const path = require('path')
+const chokidar = require('chokidar')
+
+const buildConfig = require('./config.json')
+const { paths } = buildConfig
+
+const appSassPath = path.join(paths.assets, 'sass')
+const docsSassPath = path.join(paths.docsAssets, 'sass')
+const v6SassPath = path.join(paths.v6Assets, 'sass')
+
+const appCssPath = path.join(paths.public, 'stylesheets')
+const v6CssPath = path.join(paths.public, 'v6', 'stylesheets')
+
+const v6PublicPath = path.join(paths.public, 'v6')
+
+function generateAssets () {
+  clean()
+  sassExtensions()
+  sass(appSassPath, appCssPath)
+  sass(docsSassPath, appCssPath)
+  sass(v6SassPath, v6CssPath)
+  copyAssets(paths.assets, paths.public)
+  copyAssets(paths.docsAssets, paths.public)
+  copyAssets(paths.v6Assets, v6PublicPath)
+}
+
+function runTasks () {
+  generateAssets()
+  watch()
+  runNodemon()
+}
+
+function clean () {
+  // doesn't clean extensions.scss, should it?
+  del.sync(['public/**', '.port.tmp'])
+}
+
+function sassExtensions () {
+  const extensions = require('../lib/extensions/extensions')
+  let fileContents = '$govuk-extensions-url-context: "/extension-assets"; '
+  fileContents += extensions.getFileSystemPaths('sass')
+    .map(filePath => `@import "${filePath.split(path.sep).join('/')}";`)
+    .join('\n')
+  fs.writeFileSync(path.join('lib', 'extensions', '_extensions.scss'), fileContents)
+}
+
+function sass (sassPath, cssPath) {
+  if (!fs.existsSync(sassPath)) return
+  console.log('compiling CSS...')
+  const sass = require('sass')
+  fs.mkdirSync(cssPath, { recursive: true })
+  fs.readdirSync(sassPath).forEach(file => {
+    if (!file.endsWith('.scss')) return
+
+    try {
+      const result = sass.renderSync({
+        file: `${sassPath}/${file}`,
+        logger: sass.Logger.silent,
```

Comment:
This shouldn't be needed, `quietDeps: true` should be sufficient to silence warnings from GOV.UK Frontend Sass

Reply:
Done